askama_derive/Cargo.toml

```diff
@@ -31,6 +32,7 @@ with-tide = []
 with-warp = []
 
 [dependencies]
+blake3 = "1.4.0"
```

Comment:
Could you benchmark your PR with `blake3` and `ahash`? I guess `blake3` is faster, but `ahash` is much more likely to already be a dependency in your `cargo tree`. It's a non-optional dependency of `actix-web` and `sqlx-core`.

Reply:
I think `ahash` will be faster actually, but I think using blake3 or any cryptographic hash function is preferable since we'd like a very stable, collision-free hash for the file names since we can't deal with collisions. `ahash` and others like it are meant more for use in a hashmap, but maybe that's fine?

Reply:
Yeah, I think blake3 is the better choice for this kind of thing.